app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', class_name: 'Post'
+  has_many :comments, foreign_key: 'author_id', class_name: 'Like'
+  has_many :likes, foreign_key: 'author_id', class_name: 'Comment'
```

Comment:
✅ The associations between models are set up correctly according to the ERD diagram 👍🏻